vars/githubscm.groovy

```diff
@@ -117,21 +117,38 @@ def forkRepo(String credentialID = 'kie-ci') {
 }
 
 def createPR(String pullRequestTitle, String pullRequestBody = '', String targetBranch = 'master', String credentialID = 'kie-ci') {
+    def pullRequestLink
+    try {
+        pullRequestLink = executeHub("hub pull-request -m '${pullRequestTitle}' -m '${pullRequestBody}' -b '${targetBranch}'", credentialID)
+    } catch (Exception e) {
+        println "[ERROR] Unable to create PR. Please make sure the targetBranch ${targetBranch} is correct."
+        throw e;
+    }
+    println "Please see the created PR at: ${pullRequestLink}"
+    return pullRequestLink
+}
+
+def createPRWithLabels(String pullRequestTitle, String pullRequestBody = '', String targetBranch = 'master', String[] labels, String credentialID = 'kie-ci') {
```

Comment:
just notice, please have the labels as second parameter (as it is not optional)

Reply:
btw, why do we need a `String[]` for the labels ? cant we have a simple `List` ?